src/storage/user_storage.rs

```diff
@@ -0,0 +1,154 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use std::{
+    collections::{HashMap, HashSet},
+    path::PathBuf,
+    sync::{Arc, Mutex},
+};
+
```

Comment:
Remove newline ?

Reply:
I can but we don't have any format rule on import grouping (still unstable)